geomesa-core/src/test/scala/org/locationtech/geomesa/core/data/FeatureWritersTest.scala

```diff
@@ -103,7 +103,6 @@ class FeatureWritersTest extends Specification {
 
         /* write the feature to the store */
         fs.addFeatures(featureCollection)
-        fs.flush()
```

Comment:
I've lost track of when we want to flush.  I suppose this does not matter since it is using MockAccumulo?